app.js

```diff
@@ -34,3 +34,52 @@ var stuff = [
 //   // console.log(stuff[i].name);
 //   // console.log(stuff[i].src);
 // }
+
+function findRandom(){
```

Comment:
Since we worked on this together, you should write some comments to yourself so that you remember what this is doing.